docs/app/templates/getting-started/assets.hbs

```diff
@@ -27,22 +27,28 @@
 <h2 id="importing-css-assets">Importing static CSS Assets</h2>
 
 <p>
-  When not using a CSS preprocessor (see below) ember-bootstrap will
-  automatically import the static Bootstrap CSS into your ember-cli build. In
-  situations where you prefer to use another strategy for importing Bootstrap
-  CSS, you can opt out of CSS import by setting the
-  <code>importBootstrapCSS</code>
-  option to false in your
-  <code>ember-cli-build.js</code>:
+  When not using a CSS preprocessor (see below) you will need to import the
+  static Bootstrap CSS into yourself.
 </p>
+<p>
+  For classic apps, add the following lines to
+  <code>ember-cli-build.js</code>
 
-<div>
+  {{! prettier-ignore }}
   <pre class="highlight"><code>//your-bootstrap-app/ember-cli-build.js
-      module.exports = function(defaults) { let app = new EmberApp(defaults, {
-      'ember-bootstrap': { 'importBootstrapCSS': false } }); return
-      app.toTree(); };
-    </code></pre>
-</div>
+module.exports = function(defaults) {
+  let app = new EmberApp(defaults, { /* Configuration */ });
+  app.import('node_modules/bootstrap/dist/css/bootstrap.css');
+  app.toTree();
+};</code></pre>
+</p>
+
+<p>
+  For Embroider apps, import the CSS file in
```

Comment:
This is not depending on the build system, because both patterns work both under classic and Embroider. It's more about legacy vs. next-gen pattern IMHO. But we can tweak this later...